### DISCUSSION TITLE
Collections: book x of y.

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#203] [Ideas]
observation : when a series of books are to be read in order, it would be interesting to know the number in the collection of the title.

UNIMARC : 461$a (collection title) and 461$v (number) ;  the famous : "No one knows what 461$v means. That's also silly. #201" :-) 
and 
MARC21 : 490$a (collection title) and 490$v (number) :-)
source marc21 https://www.marc21.ca/M21/COD/ZONE/BIB/BCZ-4XX.html#G490 (english and french)

Suggestion: have 2 additional fields 
    -1 for "Collection name"
    -1 for "Game number".

I join a sqlite database with 7 books "Harry Potter collection" and the 3 books of Liu Cixin - "The Three Body Problem collection".

[collection-of-books-to-read-in-order.sqlite.tar.gz](https://github.com/textbrowser/biblioteq/files/8451765/collection-of-books-to-read-in-order.sqlite.tar.gz)

for the librarian, knowing that the book is part of a collection and that the member must be told to read them in order seems to me an interesting option to propose.

When do you think?

**Comment by textbrowser** [comment]
Sure, good idea.

**Reply by meteos77** [reply]
it is always a prospect of improvement
